server/Dockerfile

```diff
@@ -3,24 +3,34 @@
 FROM aiplanning/planutils:latest
 
 # FD Planner
-RUN planutils install -f -y lama-first 
+RUN planutils install -f -y lama-first
 
 # Satisficing classical planning
 RUN planutils install -f -y dual-bfws-ffparser
 
 #Optimal classical planning
 RUN planutils install -f -y delfi
 
-# top-k classical planning
-RUN planutils install -f -y forbiditerative-topk
+# top-k classical planning -- currently unavailable
+# RUN planutils install -f -y forbiditerative-topk
```

Comment:
Is there an issue we should track from planutils pointing to what's wrong with the top-k planner?